api/api.go

```diff
@@ -40,8 +55,28 @@ func Setup(ctx context.Context, r *mux.Router, cognitoClient cognito.Client, use
 
 	r.HandleFunc("/tokens", api.TokensHandler(ctx)).Methods("POST")
 	// self used in paths rather than identifier as the identifier is JWT tokens passed in the request headers
-	r.HandleFunc("/tokens/self", api.SignOutHandler(ctx)).Methods("DELETE")
+	r.HandleFunc("/tokens/self", baseHandler(api.SignOutHandler).ServeHTTP).Methods("DELETE")
 	r.HandleFunc("/tokens/self", api.RefreshHandler(ctx)).Methods("PUT")
 	r.HandleFunc("/users", api.CreateUserHandler(ctx)).Methods("POST")
 	return api, nil
 }
+
+func WriteErrorResponse(ctx context.Context, w http.ResponseWriter, errorResponse *models.ErrorResponse) {
```

Comment:
I haven't looked closely, but does this function actually need to be exported?

Reply:
No it doesn't, that is leftover from where it used to be in a different package.
I'll update it